### PR TITLE
Added toast component

### PR DESCRIPTION
**Pull request by athar05**

### CODE REVIEW

**Comment by athar05** [comment]
Live Preview: https://deploy-preview-12--aquaui.netlify.app/components.html

**Review by sashaboi — APPROVED** [review]
Looks good ! I like the styling of the toast !

File: components.html [hunk 1]
```diff
@@ -1167,15 +1168,53 @@ <h3 class="section-sub-heading">Styles</h3>
...
+                  x.className = x.className.replace("show", "");
+                }, 3000);
+              }
+
+            </code>
+          </pre>
+
+        </div>
+
+
```
Comment (Balakumar3018) [inline components.html:1205]: remove extra spaces

File: style.css [hunk 2]
```diff
@@ -1059,6 +1060,28 @@ h6 {
...
+  z-index: 1;
+  left: 50%;
+  bottom: 30px;
+}
+
+#toast-component.show {
+  visibility: visible;
+}
+
+/* Toast section ends  */
```
Comment (Balakumar3018) [inline style.css:1083]: remove commented codes

**Review by Balakumar3018 — APPROVED** [review]
Good to merge 
Left few suggestions

**Comment by athar05** [comment]
> Looks good ! I like the styling of the toast !

Thank you

**Pull request merged**